.gitignore

```diff
@@ -43,6 +43,5 @@ html_docs
 
 # random old stuff that we should look at the necessity of...
 /tmp/
-backwards/
```

Comment:
hmm... maybe we need a different solution then - the problem that with this line the [this file](https://github.com/elastic/elasticsearch/pull/22185/files#diff-75d03fba77ecf12a877899dec90484b3) was ignored by git.